engine-transactions/src/lib.rs

```diff
@@ -1,5 +1,8 @@
-use crate::prelude::types::{Address, Wei};
-use crate::prelude::{vec, TryFrom, Vec, H160, U256};
+#![cfg_attr(not(feature = "std"), no_std)]
+#![cfg_attr(not(feature = "std"), feature(alloc_error_handler))]
```

Comment:
Should be able to delete this too I think

Reply:
I can't tests without that:
```
error[E0152]: found duplicate lang item `panic_impl`
  --> engine/src/lib.rs:36:1
   |
36 | pub unsafe fn on_panic(info: &::core::panic::PanicInfo) -> ! {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: the lang item is first defined in crate `std` (which `aurora_engine_transactions` depends on)
   = note: first definition in `std` loaded from /Users/anymacstore/.rustup/toolchains/nightly-2021-08-01-x86_64-apple-darwin/lib/rustlib/wasm32-unknown-unknown/lib/libstd-ccdba070b604d7bc.rlib
   = note: second definition in the local crate (`aurora_engine`)

error[E0152]: found duplicate lang item `oom`
  --> engine/src/lib.rs:59:1
   |
59 | pub unsafe fn on_alloc_error(_: core::alloc::Layout) -> ! {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: the lang item is first defined in crate `std` (which `aurora_engine_transactions` depends on)
   = note: first definition in `std` loaded from /Users/anymacstore/.rustup/toolchains/nightly-2021-08-01-x86_64-apple-darwin/lib/rustlib/wasm32-unknown-unknown/lib/libstd-ccdba070b604d7bc.rlib
   = note: second definition in the local crate (`aurora_engine`)

For more information about this error, try `rustc --explain E0152`.
error: could not compile `aurora-engine` due to 2 previous errors
```